Automattic/pocket-casts-ios
# put podcast author in Artist "now playing" fields

Fixes #48 

This PR changes the "Now Playing" field mappings to the proper fields to ensure Siri says the correct details, and Siri Suggestions has accurate information to make meaningful suggestions (as suggested in the [MPNowPlayingInfoCenter](https://developer.apple.com/documentation/mediaplayer/mpnowplayinginfocenter) doc)

The date was put in place of the author as a convenience in the past. Quote from an old issue "gives you more relevant info in that small space". I think if we want this information in the Now Playing widget it might be worth asking Apple for a "PublishedDate" field in the api (not sure how likely this is though).

I also compared Now Playing on an android device (just visually) and date doesn't appear to be pushed to the Now Playing widget there either (at least on a OnePlus device). cc @geekygecko maybe you can confirm if it does and if this should be consistent between the 2 platforms?

| before | after |
| --- | --- |
| ![IMG_0015](https://user-images.githubusercontent.com/4081020/213541998-d5eda826-d535-4258-a7fe-effe9880cb76.PNG) | ![IMG_0014](https://user-images.githubusercontent.com/4081020/213542070-90b5c087-9eb6-4f73-90be-74075daae996.PNG) |

_Screenshots are from iOS 15. I don't have an iOS 16 device to dev with but from seeing the Now Playing widget in iOS 16 it seems to show less info on the lock screen, so I think this change is even more relevant now since you only get `Artist` showing in the widget._

## To test

1. pull pr and deploy on a real device (simulator doesn't show now playing info)
2. start playing a podcast
3. ask siri "what is playing", it should say something like "[podcast title] by [podcast author]" instead of "... by [podcast date]"
4. if you have a car with bluetooth, pair your phone with the car and check that your car's "now playing" display info shows the podcast author instead of the date

## Checklist
